packages/daisyui/src/templates/FieldHelpTemplate/FieldHelpTemplate.tsx

```diff
@@ -14,7 +15,17 @@ export default function FieldHelpTemplate<
   S extends StrictRJSFSchema = RJSFSchema,
   F extends FormContextType = any,
 >(props: FieldHelpProps<T, S, F>) {
-  const { help } = props;
+  const { help, uiSchema = {}, registry } = props;
+  const uiOptions = getUiOptions<T, S, F>(uiSchema, registry?.globalUiOptions);
```

Comment:
Make this patterned after the `FieldDescriptionTemplate` in this theme using the `RichHelp` component mentioned in the comment in `core`